### PR TITLE
[Doc] Fix the error description for the number of bytes of double type.

### PR DESCRIPTION
**Pull request by jollykingCN**
Fix the error description of double type: 12 bytes is modified to 8 bytes

### CODE REVIEW

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

File: docs/zh-CN/sql-reference/sql-statements/Data Definition/CREATE TABLE.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ under the License.
             范围：-2^127 + 1 ~ 2^127 - 1
         FLOAT（4字节）
             支持科学计数法
-        DOUBLE（12字节）
+        DOUBLE（8字节）
```
Comment (EmmyMiao87) [inline docs/zh-CN/sql-reference/sql-statements/Data Definition/CREATE TABLE.md:69]: Please change the title of PR in English.
Also the en/sql-relerence/sql-statements/Data Defination/CREATE TABLE.md shoudl be changed.

**Review by EmmyMiao87 — CHANGES_REQUESTED** [review]

File: docs/zh-CN/sql-reference/sql-statements/Data Definition/CREATE TABLE.md (hunk 1 above)
Comment (jollykingCN) [reply docs/zh-CN/sql-reference/sql-statements/Data Definition/CREATE TABLE.md:69]: Done

**Review by jollykingCN — COMMENTED** [review]
Done

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**